eslint.config.mjs

```diff
@@ -824,14 +824,6 @@ export default typescript.config([
       'testing-library/no-unnecessary-act': 'off',
     },
   },
-  {
-    name: 'testing-library/react - tsx files',
-    files: ['**/*.spec.{tsx,jsx}', 'tests/js/**/*.{tsx,jsx}'],
```

Comment:
The rule above lists all files, and has no exceptions for rules that we want enabled:

```
name: 'testing-library/react',
    files: ['**/*.spec.{ts,js,tsx,jsx}', 'tests/js/**/*.{ts,js,tsx,jsx}'],
```